language/stdlib/modules/LBR.move

```diff
@@ -1,51 +1,78 @@
 address 0x1 {
+/// This module defines the `LBR` currency as an on-chain reserve. The
+/// `LBR` currency differs from other currencies on-chain, since anyone can
+/// "atomically" swap into, and out-of the `LBR` as long as they hold the
+/// underlying currencies. This is done by specifying the makeup of, and
+/// holding the reserve of backing currencies for the `LBR` on-chain.
+/// Users can create `LBR` coins by passing in the backing
+/// currencies, and can likewise "unpack" `LBR` to get the backing coins
+/// for that coin. The liquidity of the reserve is enforced by the logic in
+/// this module that ensures that the correct amount of each backing currency
+/// is withdrawn on creation of an `LBR` coin, and that only the appropriate
+/// amount of each coin is returned when an `LBR` coin is "unpacked."
 
 module LBR {
-    use 0x1::CoreAddresses;
     use 0x1::Coin1::Coin1;
     use 0x1::Coin2::Coin2;
+    use 0x1::CoreAddresses;
     use 0x1::FixedPoint32::{Self, FixedPoint32};
     use 0x1::Libra::{Self, Libra,  RegisterNewCurrency};
-    use 0x1::Signer;
     use 0x1::Roles::{Capability, TreasuryComplianceRole};
+    use 0x1::Signer;
 
-    // The type tag for this coin type.
+    /// The type tag representing the `LBR` currency on-chain.
     resource struct LBR { }
 
-    // A reserve component holds one part of the LBR. It holds
-    // both backing currency itself, along with the ratio of the backing
-    // asset to the LBR (i.e. 1Coin1 to 1LBR ~> ratio = 1).
+    /// A `ReserveComponent` holds one part of the on-chain reserve that backs
+    /// `LBR` coins. Each `ReserveComponent` holds both the backing currency
+    /// itself, along with the ratio of the backing currency to the `LBR` coin.
+    /// For example, if `Coin1` makes up 1/2 of an `LBR`, then the `ratio` field would be 0.5.
     resource struct ReserveComponent<CoinType> {
-        // Specifies the relative ratio between `CoinType` and LBR (i.e. how
-        // many `CoinType`s makeup one LBR).
+        /// Specifies the relative ratio between the `CoinType` and `LBR` (i.e. how
+        /// many `CoinType`s makeup one `LBR`).
```

Comment:
"make up" should be two words here, and while you're at it, there should be a commas after "i.e." and "e.g." throughout the document.